frontend/benefit/applicant/jest.config.js

```diff
@@ -1,5 +1,20 @@
 const sharedConfig = require('../../jest.config.js');
 module.exports = {
   ...sharedConfig,
-  rootDir: '../../',
```

Comment:
It was a bad idea to read jest configs from root directory because then all the tests were run instead of subprojects own tests. Now fixed.